src/VirtoCommerce.Platform.Caching/Redis/RedisPlatformMemoryCache.cs

```diff
@@ -1,47 +1,90 @@
 using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
 using Microsoft.Extensions.Caching.Memory;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
-using Polly;
-using Polly.Retry;
 using StackExchange.Redis;
 using VirtoCommerce.Platform.Caching;
+using VirtoCommerce.Platform.Core.Caching;
 using VirtoCommerce.Platform.Core.Common;
 
 namespace VirtoCommerce.Platform.Redis
 {
     public class RedisPlatformMemoryCache : PlatformMemoryCache
     {
         private readonly ISubscriber _bus;
```

Comment:
_bus can be converted to local var

Reply:
Not after code changes, it's used in `EvictionCallback`